dbms/src/DataStreams/CreatingSetsBlockInputStream.cpp

```diff
@@ -238,13 +238,6 @@ void CreatingSetsBlockInputStream::createOne(SubqueryForSet & subquery)
             }
         }
 
-
-        if (subquery.join)
-        {
-            FAIL_POINT_TRIGGER_EXCEPTION(FailPoints::exception_mpp_hash_build);
```

Comment:
`extern const char exception_mpp_hash_build[];` in this file can also be deleted.

Reply:
Done.